xrpl/models/amounts/mpt_amount.py

```diff
@@ -49,3 +49,16 @@ def to_currency(self: Self) -> MPTCurrency:
             The MPTCurrency for this MPTAmount.
         """
         return MPTCurrency(mpt_issuance_id=self.mpt_issuance_id)
+
+
+class MPTIssue:
```

Comment:
You can delete this and just use the existing `MPTCurrency` model.

Reply:
thanks, I didn't realize that there is an existing definition.